resolwe/flow/executors/__main__.py

```diff
@@ -81,5 +81,8 @@ async def _close_tasks(pending_tasks, timeout=5):
     loop = asyncio.get_event_loop()
     configure_logging()
     loop.run_until_complete(_run_executor())
-    loop.run_until_complete(_close_tasks(asyncio.Task.all_tasks()))
+    # TODO: remove asyncio.Task.all_tasks when we stop supporting
+    # Python 3.6.
+    all_tasks = getattr(asyncio, "all_tasks", None) or asyncio.Task.all_tasks
```

Comment:
+1, this will probably solve @romunov's problem with too many deprecation logs.

Reply:
I have done it since `asyncio.Tasks.all_tasks` is not only deprecated but actually removed in Python 3.9.
Great that it solves more than one problem :)